xhprof_lib/config.sample.php

```diff
@@ -8,10 +8,11 @@
 $_xhprof['dbpass'] = 'password';
 $_xhprof['dbname'] = 'xhprof';
 $_xhprof['dbadapter'] = 'Pdo';
-$_xhprof['servername'] = 'myserver';
+$_xhprof['servername'] = 's01';
 $_xhprof['namespace'] = 'myapp';
 $_xhprof['url'] = 'http://url/to/xhprof/xhprof_html';
 $_xhprof['getparam'] = "_profile";
+$_xhprof['displayparam'] = "_display";
```

Comment:
This is an enhancement of a feature already present, namely the option to enable/disable displaying of the xhprof URL for the current request in the output. But many URLs (particularly on projects using a frontend framework like angular, ember, react, etc.) break with enabled 'display', particularly with JSON output. I thought it would be a nice thing to be able to switch this on/off at runtime, via a parameter - exactly like for the _profile toggle.